lib/razor/config.rb

```diff
@@ -64,9 +64,10 @@ def initialize(env, fname = nil, defaults_file = nil)
         raise InvalidConfigurationError,
           _("The configuration file %{filename} does not exist") % {filename: fname} if @values.empty?
       rescue Errno::EACCES
-        raise InvalidConfigurationError,
-          _("The configuration file %{filename} is not readable") % {filename: fname}
+        # The given file is not readable. This is okay.
```

Comment:
I think this should actually throw an error. Anytime a referenced file isn't readable via permissions, I'd expect this to throw.

Reply:
This achieves the equivalent effect of the `readable?` check -- if the file is not readable in the original code, it is a noop just like it is here.

Reply:
Not exactly, since `readable?` also returns `true` if the file does not exist. If the file exists, and can't be read, my intuition tells me that should be an error.

Reply:
Oh ok, if that's what we want to do then I updated the PR to reflect that.